src/NuGet.Core/NuGet.Protocol/HttpSource/HttpRetryHandler.cs

```diff
@@ -93,10 +93,6 @@ public async Task<HttpResponseMessage> SendAsync(
                     // In many cases due to the external retry loop, this will be always be 1 * request.RetryDelay.TotalMilliseconds + 0-200 ms
                     if (_enhancedHttpRetryHelper.IsEnabled)
                     {
-                        if (tries >= 3 || (tries == 0 && request.IsRetry))
```

Comment:
This was used as logging to ensure that the retry helper is worth it, basically a comparison between old and new, but given we've been working with it for a few years, I think the answer is yes, so we don't need this.